src/Pyz/Zed/ContactUs/Business/ContactUsWriter/ContactUsWriter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Pyz\Zed\ContactUs\Business\ContactUsWriter;
+
+use Generated\Shared\Transfer\ContactUsTransfer;
+use Generated\Shared\Transfer\PyzContactUsEntityTransfer;
+use Pyz\Zed\ContactUs\Persistence\ContactUsEntityManagerInterface;
+
+class ContactUsWriter implements ContactUsWriterInterface
+{
+    protected $contactUsEntityManager;
+
+    public function __construct(ContactUsEntityManagerInterface $contactUsEntityManager)
+    {
+        $this->contactUsEntityManager = $contactUsEntityManager;
+    }
+
+    /**
+     * @param ContactUsTransfer $contactUsTransfer
+     *
+     * @return PyzContactUsEntityTransfer
+     */
+    public function saveContactUsData(ContactUsTransfer $contactUsTransfer): PyzContactUsEntityTransfer
+    {
+        $contactUsEntityTransfer = new PyzContactUsEntityTransfer();
```

Comment:
we don't use persistent level entities in business level. WE should keep Entities ONLY in PERSISTENCE level.

please use Factory to create query and get entity from there before mapping. And do it in EntityManager